_includes/header.html

```diff
@@ -6,6 +6,9 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="brand-heading">Konstructs</h1>
                         <p class="intro-text">An open source Infiniminer/Minecraft inspired multiplayer game.</p>
+                        <p>
+                          <iframe width="280" height="158" src="https://www.youtube.com/embed/8HmZ1Rk0Kak" frameborder="0" allowfullscreen></iframe>
```

Comment:
It's looking "good" in portrait mode on my HTC Magic. I'd say few phones have lower resolution than that :-)